server/xpub-controller/manuscript.js

```diff
@@ -83,23 +84,10 @@ class Manuscript {
       )
     }
 
-    const originalEmails = (manuscript.getAuthor()
-      ? manuscript.getAuthor()
-      : []
-    )
-      .map(author => author.alias.email)
-      .join(',')
-
+    const oldSignature = manuscript.submitterSignature
     manuscript.applyInput(data)
-
-    const newAuthors = manuscript.getAuthor()
-    const newEmails = newAuthors.map(author => author.alias.email).join(',')
-
-    // Send email here only when author changes...
-    if (newEmails !== originalEmails) {
-      // sendEmail
-      const notify = new Notification(this.config, newAuthors)
-      notify.sendDashboardEmail()
+    if (data.submitterSignature && oldSignature !== data.submitterSignature) {
+      this.notify.sendFinalSubmissionEmail(manuscript.getAuthor())
```

Comment:
After testing, the email seems to send after you click Submit, and does not wait for the user to click Confirm on the dialog box. This is an issue since the user also has the option to click Cancel, and we wouldn't want the email sent out in this case.